### DISCUSSION TITLE
Useful end of dungeon rewards, to incentivise people running RDF

### DISCUSSION THREAD

**Discussion by KaneoHunter** [chromiecraft/chromiecraft#3395] [Suggestions]
Can we get some useful rewards at the end of what's currently called "Random Burning Crusade Dungeon"?

You can theme the rewards around various professions. The mats I list add up to around 10g, and as someone who has various alts with all the gathering professions, I could easily farm these items in less time than it takes to run a random dungeon, I would just rather queue and help players in need get gear. It's also more fun.

Worn Miner's Satchel
1x Arcane Crystal
20x Dense Stone
20x Thorium Ore

Grass-Stained Satchel
5x Dreamfoil
5x Ice Cap
5x Gromsblood

Grass-Stained Satchel II
20x Stranglekelp
20x Arthas' Tear
2x Firebloom
2x Purple Lotus

Blood-Drenched Satchel
2x Bear Flank
4x Zesty Clam Meat
4x Raw Nightfin Snapper
1x Stonescale Eel

Mystical Satchel
1x Elemental Earth
1x Elemental Fire
1x Large Brilliant Shard
2x Dream Dust

Going to avoid suggesting a Skinning bag, as Skinning has always needed all the help it can get to stay relevant.

The AH is drying up, queues need to pop faster, and I want reasons to run dungeons besides altruism. I know that in the past material rewards from dungeons didn't work out, but I believe that was more an issue of tuning/balance, than it was a bad system. Can we look at bringing it back, with an eye on having a wide variety of rewards (to not flood any single market), and a fair gold value for each bag?

Some half-baked ideas for other bags:

Reputable Satchel
*random selection of BOE +rep items for vanilla factions*

Making these BOE would add a whole new gold-sink where older players can boost rep on their alts, and give newer players much welcome gold. 

Bulging Satchel
10g

Some very welcome gold for newer players, allowing them to buyout the contents of another bag, while increasing the variety of bags available, reducing how many mats are entering the economy. I don't think 10g will inflate the economy too much, and it will probably be one of the least-desired bags.

**Comment by Wayran-is-typing** [comment]
Could be a great idea!! RDF is a little dead at 55+ if you are dps and a random/lonely player who wants chill on dungeons

**Comment by Amrx2** [comment]
Hi KaneoHunter, thanks for the suggestion. This is something we actually trialed out for a while when we moved into the 50-59 bracket, didn't go well at all and we reverted the change to gold and then gear as well again. Usual concerns about the economy, thousands of those mats flooding onto the market etc. Will watch the reaction of the community of course. Cheers.

**Reply by KaneoHunter** [reply]
The obvious opposition was always going to be the impact on the market.

It's worth noting that we're in a different "stage" of the game now, I don't have the number but I'm willing to bet that the number of RDF runs going on now are significantly lower than in the 50-59 bracket, additionally, more people have alts with Herbing/Mining to gather for themselves.

I'd like to outline some problems I'm trying to solve, so we can collaborate to get a solution.

1. Low liquidity of high demand materials.

I think this has been caused by a number of things, more alts allowing people to gather/craft for themselves has reduced demand, while raw gold farms becoming more viable with the new level cap and raid gear reducing supply. Important items like Dreamfoil, Bear Flank, Zesty, Clam Meat, Raw Nightfin Snapper, and Stranglekelp (to name a few) are not frequently available on the market.

2. Incentivise fun content

Maybe a more personal one, but with how popular GDKP is on larger servers, as well as tank/healing "services", I can't be the only person who enjoys running group content to make gold. I think more people would have something fun to do, and a reason to log in, if they were given an incentive to do the content they enjoy. 

3. Slow RDF queues outside peak times, and speed up DPS queues.

It's a tale as old as time in any MMO. Tanks and healers are in high demand, they get runs faster, gear faster, then stop running the content, exacerbating their scarcity. In TBC an attempt was made to fix this, with nethers and bis gems from dungeon bosses, as well as the badge system. In WOTLK they added orbs (which were the new nethers, and could be turned in to dungeon mats), and then added a weekly-capped badge that slowed down how quickly tanks/healers completed the content, keeping queues going for longer. Unfortunately, vanilla/classic content doesn't have anything like this, which is why I want to add something to act similarly.

4. Difficult-to-enter economy

With the low liquidity in the market, and the high amount of gold in the economy for our server, it's hard to get going on your first character. In addition to new gold-sinks, it would be great if we had more ways to funnel gold from veteran players down to the newer players. To elaborate, you can't always buy the mats/consumables you need to raid, and when they are available it's often overpriced as people have so much gold to throw at items, your only option is to gather everything for yourself, which has worked for me, but I can't imagine this works for a more casual player.

Would love to hear more people's takes on solutions to these problems.
